recipes/loos/build.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+set -xe
+
+cmake CMakeLists.txt -DCMAKE_INSTALL_PREFIX="$PREFIX" -DPython3_EXECUTABLE="$PYTHON" -DCMAKE_INSTALL_LIBDIR=lib -DBUILD_SHARED_LIBS=ON -DCMAKE_BUILD_TYPE=Release
```

Comment:
Please use `CMAKE_ARGS` to pass on the correct settings.
```suggestion
cmake CMakeLists.txt ${CMAKE_ARGS} -DPython3_EXECUTABLE="$PYTHON" -DBUILD_SHARED_LIBS=ON -DCMAKE_BUILD_TYPE=Release
```